### DISCUSSION TITLE
Upgrade not detected

### DISCUSSION THREAD

**Discussion by sebastienserre** [FreshRSS/FreshRSS#5163] [Q&A] [answered]
**Describe the bug**
The update features doesn't detect the 1.21.0 version

**To Reproduce**
1. Go to settings / update
2. Click on update button
3. See error

**Screenshots**
![2023-03-05_10-01](https://user-images.githubusercontent.com/3902621/222951448-438eedb9-26e4-4ab9-a509-7f4f5e013455.jpg)

**Comment by Alkarex** [comment, marked as answer]
Hello,
The ZIP-based update using the Web interface is always some days late, as it requires a manual process from @marienfressinaud (until we rewrite the mechanism https://github.com/FreshRSS/FreshRSS/pull/1760 ). It is also partially on purpose to potentially address regressions caught by users of faster release systems (git, Docker).

**Reply by marienfressinaud** [reply]
I think I'll enable the updates to 1.21 ~Wednesday btw

**Reply by marienfressinaud** [reply]
Done :heavy_check_mark:
